src/components/Firestore/Requests/RequestDetails/CodeViewer/index.scss

```diff
@@ -81,3 +85,140 @@
     width: 61px !important;
   }
 }
+
+/* CodeMirror theme, based on http://go/firebase-console-codemirror-theme */
```

Comment:
Was the go link intentional?

Reply:
Yeah, I created the go-link just for this comment